templates/hierarchy/piechart.js

```diff
@@ -0,0 +1,45 @@
+function drawChart(chartData) {
```

Comment:
To keep it DRY, Could we use the existing `piechart.js` from bootstrap directory?